server/settings/publish_plugins.py

```diff
@@ -770,6 +797,10 @@ class PublishPuginsModel(BaseSettingsModel):
         default_factory=ValidateBaseModel,
         title="Validate Version"
     )
+    ValidateContainers: ValidateContainersModel = SettingsField(
+        default_factory=ValidateContainersModel,
+        title="Validate Containers"
```

Comment:
Should we update the title to `Validate Outdated Containers` so it matches the label on the plug-in itself? 

With just **Validate Containers** it leaves me a bit guessing _what_ it validates about the containers?

Reply:
Eh, why you wrote it after merge, now I have to create new PR...

Reply:
Sorry, day off today - saw it a bit too late.